lib/client/onNotification.js

```diff
@@ -36,34 +36,50 @@ exports.func = function (args) {
       args.jar = { session: args.jar }
     }
     const session = getSession({ jar: args.jar })
-    const client = new SignalR('wss://realtime.roblox.com/notifications', ['usernotificationhub'], 3, true) // wss for https
-    client.headers.Cookie = '.ROBLOSECURITY=' + session + ';'
-    client.on('usernotificationhub', 'notification', function (name, message) {
-      notifications.emit('data', name, JSON.parse(message))
-    })
-    notifications.on('close', client.end)
-    client.serviceHandlers.connectFailed = function (err) {
-      notifications.emit('error', new Error('Connection failed: ' + err.message))
-      if (retries !== -1) {
-        if (retries > max) {
-          notifications.emit('close', new Error('Max retries reached'))
-        } else {
-          setTimeout(connect, 5000, retries + 1)
+    let userNotificationConnection = null;
```

Comment:
Why is this instantiated and set to null ? could it just be set directly as on line 41?

Reply:
It is set to null to declare it. If you test the changes I did for the realtime API you will see that it works.